crates/wasi-provider/src/wasi_runtime.rs

```diff
@@ -299,9 +299,20 @@ impl WasiRuntime {
             let func = match export {
                 wasmtime::Extern::Func(f) => f,
                 _ => {
-                    return Err(anyhow::anyhow!(
-                    "_start import was not a function. This is likely a problem with the module"
-                ))
+                    let message = "_start import was not a function. This is likely a problem with the module";
```

Comment:
This was the one Err in this method that did not notify the channel.

Reply:
Good catch. I swore I updated this to the new notify logic...